src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -215,7 +215,7 @@ internal void SendAsynchronously(IEnumerable<IWorkspaceModel> workspaces)
                     {
                         State = UploadState.Succeeded;
                         Error = UploadErrorType.None;
-                        CustomizerURL = String.Concat(serverUrl, serverResponce.Value);
+                        CustomizerURL = String.Concat(serverUrl, ":", port, serverResponce.Value);
```

Comment:
Just to clarify, the port isn't required unless it is specified in the config file. By default, http is port 80 and https is 443. In those cases, the port should not be supplied and should not appear in the customizer url. 

My recommendation is that we remove the url from the App.config entirely. So, now the url specified in the app.config should be simply:

http://10.39.165.198:3000

The prefix (http:// in this case) and the port are a part of the URL.